tools/cdb2sql/cdb2sql.cpp

```diff
@@ -161,7 +161,8 @@ void cdb2sql_usage(int exit_val)
 }
 
 
-const char *words[] = {
+const char *level_one_words[] = {
+  "@desc", "@ls", "@send",
```

Comment:
Perhaps not show these until "@ + TAB" ?

Reply:
sure, but then how do you know to use '@' ?

Reply:
Should i show '@' as the first word though?